### PR TITLE
Enable C++ bindings

### PR DESCRIPTION
**Pull request by astrofrog**
The C++ bindings were not enabled

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -13,7 +13,9 @@ fi
 
 
 ./configure --prefix=$PREFIX \
-            --disable-dependency-tracking
+            --disable-dependency-tracking \
+            --enable-mpi-cxx \
+            --enable-mpi-fortran
```
Comment (astrofrog) [inline recipe/build.sh]: I added the enable fortran flag to make sure it is explicit (but enabled is the default if a fortran compiler is present, which it needs to be here)

**Comment by astrofrog** [comment]
@ocefpaf @bekozi @minrk @msarahan - this is ready for review!

**Comment by ocefpaf** [comment]
Awesome! Thanks @astrofrog!!

**Comment by astrofrog** [comment]
If only we could move this fast with HDF5! ;)

**Pull request merged**